layouts/partials/pagination.html

```diff
@@ -0,0 +1,45 @@
+{{ $pag := $.Paginator }}
+{{ if gt $pag.TotalPages 1 -}}
+
+  <div class="row flex-center margin-top-large">
+ 
+  {{ with $pag.First -}}
+    <a href="{{ .URL }}" class="paper-btn">&laquo;&laquo;</a>
+  {{ end -}}
+    <a {{ if $pag.HasPrev }}href="{{ $pag.Prev.URL }}"{{ end }} class="paper-btn{{ if not $pag.HasPrev }} disabled{{ end }}">&laquo;</a>
+  {{- $ellipsed := false -}}
+  {{- $shouldEllipse := false -}}
```

Comment:
`$ellipsed` and `$shouldEllipse` seem to do the same thing. Are they both needed? Is `$shouldEllipse` sufficient?

Reply:
I did a bit of digging, as this is heavily based on Hugo's default pagination template.  Looking at the [blame report for the pagination template](https://github.com/gohugoio/hugo/blame/master/tpl/tplimpl/embedded/templates/pagination.html) it seems that whole section of code was adding in the same commit upstream.  I honestly don't quite see the need for `$ellipsed` myself, I do worry I might be missing something from the source template.  If you want I could remove all the `$ellipsed` references and verify it continues to work correctly.  Otherwise I like the idea of trying to stick as close to the internal pagination template as possible, and leaving that variable in place.

Reply:
Ah, as someone who has never implemented pagination, did not realize your code was adapted from the built in template. In that case, it's fine.